deepchem/data/data_loader.py

```diff
@@ -2014,18 +2012,13 @@ class BAMLoader(DataLoader):
     or MacOS X. To use Pysam on Windows, use Windows Subsystem for Linux(WSL).
     """
 
-    def __init__(self,
-                 featurizer: Optional[Featurizer] = None,
-                 get_pileup: Optional[bool] = False):
+    def __init__(self, featurizer: Optional[Featurizer] = None):
```

Comment:
This flag was added post the last major release so OK to remove before the next release